socrates/lib/registration/views/participate.jade

```diff
@@ -15,8 +15,13 @@ block content
     if(registrationTuple.duration === 'waitinglist')
       h1 Waitinglist Registration<br>
     else
-      h1 Registration<br>
-      small Personal Information
+      h1 Registration #{' '}
+        small Personal Information
+      
+    p.
+      You have #{expiresIn} minutes to complete your registration. The reservation will expire 
+      at #{expiresAt.locale('en-gb').format('LT')}.
```

Comment:
Du erwähnst die reservation, ohne sie vorher zu erklären. Eher andersherum: "You now have a reserved spot at the conference. <dann weiter wie oben, You have expiresIn minutes>"